jscomp/core/js_exp_make.ml

```diff
@@ -1253,6 +1259,23 @@ let rec int32_band ?comment (e1 : J.expression) (e2 : J.expression) :
 (* let int32_bin ?comment op e1 e2 : J.expression =  *)
 (*   {expression_desc = Int32_bin(op,e1, e2); comment} *)
 
+let bigint_op ?comment op (e1: t) (e2: t) = bin ?comment op e1 e2
+
+let bigint_comp (cmp : Lam_compat.comparison) ?comment (e0: t) (e1: t) =
+  bin ?comment (Lam_compile_util.jsop_of_comp cmp) e0 e1
+
+let bigint_div ~checked ?comment (e0: t) (e1: t) =
```

Comment:
looks like the option to disable checking divide by zero is really only hidden under some ancient internal-only configuration

at some point, it can be removed, perhaps in a later PR